### DISCUSSION TITLE
How does imagemagick multi -layer multi -layer and multi -layer Tifcmyk files turn transparent PNG

### DISCUSSION THREAD

**Discussion by zxc155226522** [ImageMagick/ImageMagick#6255] [Help]
### ImageMagick version

7.1.1-6

### Operating system, version and so on

win11

### Description

How does imagemagick multi -layer multi -layer and multi -layer Tifcmyk files turn transparent PNG

**Comment by fmw42** [comment]
This question is too vague. Please explain in more detail.

PNG does not support CMYK. So it has to be converted to sRGB .  PNG does not support multiple layers/pages. So each layer or page would be converted to a different output PNG. Imagemagick does not support layers in TIFF only pages as far as I know.
